### DISCUSSION TITLE
关于图片和表格位置问题

### DISCUSSION THREAD

**Discussion by Richard9797** [note286/xduts#56] [Q&A] [answered]
看了之前的讨论，您说不建议手动指定浮动体的位置。那么应该怎么控制图片和表格的位置呢。还是说自动排版就好。麻烦您了

**Comment by note286** [comment, marked as answer]
自动就好，默认就是靠上浮动，图、表、算法都是。

**Comment by sikouhjw** [comment]
> 那么应该怎么控制图片和表格的位置呢。

问题在于，『为什么需要控制』？

> 还是说自动排版就好。

对，参考 https://github.com/sikouhjw/gdutthesis/discussions/73

**Reply by Richard9797** [reply]
图片会把一段话一分为二，这种也没关系吗。

**Reply by sikouhjw** [reply]
> 图片会把一段话一分为二，这种也没关系吗。

没关系，图片在参考文献后都没关系，国外论文里多的是这样的例子，也一样能被接收。国内学术正常点的高校老师都知道 latex 图片就是这样的行为。

**Reply by note286** [reply]
文章的主体是文字，图表等都是补充，靠边站是应该的。你阅读的时候是阅读文字，看到引用了，再去附近找引用的图表等内容。

**Reply by note286** [reply]
@Richard9797 你这个一分为二，不会说的是这段文字跨页吧。

**Reply by Richard9797** [reply]
![EZJ2_Z_Y7`0K}$1LNDAH$ V](https://user-images.githubusercontent.com/65112702/222367864-ba8fca6a-8113-45df-9f73-225b2df58294.png)
例如这样的情况

**Reply by sikouhjw** [reply]
正常的情况，你不可能保证一段文字永远不分开，接受浮动体就好

**Reply by Richard9797** [reply]
嗷嗷好的，谢谢您
